test/fixtures/vcr/relay_group_add_relay.json

```diff
@@ -11,14 +11,14 @@
       "url": "http://localhost:4000/v1/token"
     },
     "response": {
-      "body": "{\"token\":{\"value\":\"w05n5iZL4peQoZ1gibP1MR1TgvSTXrIwZzCIJYRjR+0=\"}}",
```

Comment:
I mean it looks like the only things that have changed in this cassette are things like tokens, timestamps, and request IDs, all of which are incidental when we're talking about testing requests and responses. There are other cassettes where there are actual meaningful changes to things, and those should of course be kept.

It's not a huge deal, but if these changes aren't necessary, then why keep them?